client/signup/steps/theme-selection/signup-themes-list.jsx

```diff
@@ -12,48 +12,48 @@ import getThemes from 'lib/signup/themes';
 import ThemesList from 'components/themes-list';
 import { abtest } from 'lib/abtest';
 
-module.exports = React.createClass( {
-	displayName: 'SignupThemesList',
+class SignupThemesList extends Component {
 
-	propTypes: {
+	static propTypes = {
```

Comment:
We could add `translate: PropTypes.func` here, and `translate = identity` in default props.